.github/workflows/test.yaml

```diff
@@ -19,6 +19,10 @@ jobs:
         run: pip install .[testing]
       - name: Run tests
         run: pytest -ra --cov=bofire --cov-report term-missing tests
+      - name: Install optional dependencies
```

Comment:
Why not install it directly together with the whole framework above and then invoke pytest in a way that it runs also the test involving cyipopt?